# Add ability to always open dev context menu on reload button.

--- a/chrome/browser/ui/views/toolbar/reload_button.h
+++ b/chrome/browser/ui/views/toolbar/reload_button.h
@@ -42,7 +42,7 @@ class ReloadButton : public ToolbarButto
   Mode visible_mode() const { return visible_mode_; }
 
   // Enable reload drop-down menu.
-  void set_menu_enabled(bool enable) { menu_enabled_ = enable; }
+  void set_menu_enabled(bool enable) { menu_enabled_ = true; }
 
   // views::View:
   void OnThemeChanged() override;
